packages/node-client/bin/get-integration.js

```diff
@@ -4,7 +4,7 @@ const args = process.argv.slice(2);
 const nango = new Nango({ host: 'http://localhost:3003', secretKey: args[0] });
 
 nango
-    .getIntegration(args[1], args[2])
+    .getIntegrationV2(args[1])
```

Comment:
Would rather this be a separate file until we actually retire the original method